public/locales/en/panel-gettingstartedtour.json

```
@@ -0,0 +1,200 @@
+{
+  "previous-button": "Previous",
+  "next-button": "Next",
+  "finish-button": "Finish",
```

```suggestion
  "button-labels": {
    "previous": "Previous",
    "next": "Next",
    "finish": "Finish",
  }
```
?
alt. if they are recurring `common-button-labels`?

---

Ok, was a bit confused about the phrasing. Personally I think I'd prefer to not have anything names common in the files, because in general that suggest some redesign being needed

Will group as the code suggestion shows